.pre-commit-config.yaml

```
@@ -44,3 +44,10 @@ repos:
     rev: 0.5.0
     hooks:
       - id: nbstripout
+
+  - repo: https://github.com/asottile/blacken-docs
+    rev: v1.12.1
+    hooks:
+      - id: blacken-docs
+        additional_dependencies: [black==22.3.0]
+        args: [-l 120]
```

Let's disable string normalization to prevent syntax error issue and let's also consider shortening the line length for the documentation, since the code blocks are typically displayed narrower due to sidebars and other components. I was already discussing it with @engintoklu.

```suggestion
        args: [--skip-string-normalization, --line-length=88]
```

---

Alright. This will also obviate the need to ignore src files, correct?